api/src/resources/dataset.rs

```diff
@@ -122,8 +122,7 @@ pub struct NewDataset<'request> {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub description: Option<&'request str>,
 
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub has_sentiment: Option<bool>,
+    pub has_sentiment: bool,
```

Comment:
So the reason this is `Optional` is that these structs (in `api`) should accurately reflect the type of valid requests we can send to our API. In other words, maybe one day we could autogenerate these from openapi :wink: 

This shouldn't change, as our api defs aren't changing